core/src/main/java/com/nisovin/magicspells/Spell.java

```diff
@@ -911,15 +911,14 @@ protected <T extends Keyed> ConfigData<T> getConfigDataRegistryEntry(String key,
 	}
 
 	/**
-	 * @param key Path for the keys to be read from. If the path is set to something like "filter", the keys will
-	 *            be read from spell config section under the "filter" section.
+	 * @param key Path for the String or Config SpellFilter to be read from.
```

Comment:
Capitalizing string and config here is a bit odd. Also, aren't they referred to as "section-based" spell filters? All spell filters are in configs, so calling them config spell filters is a bit weird.